pandas/io/parsers.py

```diff
@@ -2281,9 +2283,10 @@ def TextParser(*args, **kwds):
         can be inferred, there often will be a large parsing speed-up.
     float_precision : str, optional
         Specifies which converter the C engine should use for floating-point
-        values. The options are None for the ordinary converter,
-        'high' for the high-precision converter, and 'round_trip' for the
-        round-trip converter.
+        values. The options are `None` or `high` for the ordinary converter,
+        `legacy` for the original lower precision pandas converter, and
+        `round_trip` for the round-trip converter.
```

Comment:
should high/legacy/round_trip have ""s to denote they are strings?

Reply:
Good point.  The options didn't have quotes before, so now with my latest commit, they do.